main/integration-tests/__test__/api-tests/common/migration/my-study-migration.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../../support/setup');
+const errorCode = require('../../../../support/utils/error-code');
+
+describe('Migration for My Studies scenarios', () => {
+  let setup;
+  let adminSession;
+  let defaultUser;
+  let defaultUserSession;
```

Comment:
Suggested renames: `targetUser` and `targetUserSession`
Reason: `default` resources are those that are already created in SWB and their IDs are provided via config file.

Reply:
Will change.